xc7/utils/prjxray_create_equiv_tiles.py

```diff
@@ -0,0 +1,1253 @@
+#!/usr/bin/env python3
+"""
+Import the top level tile interconnect information from Project X-Ray database
+files.
+
+The Project X-Ray specifies the connections between tiles and the connect
+between tiles and their sites.  prjxray_tile_import generates a VPR pb_type
+that has the correct tile pin names to connect to the routing fabric, and
+connects those tile pins to each site within the tile.
+
+"""
+
+from __future__ import print_function
+import argparse
+import os
+import sys
+import prjxray.db
+import prjxray.site_type
+import os.path
+import simplejson as json
+import re
+import sqlite3
+
+import lxml.etree as ET
+
+XI_URL = "http://www.w3.org/2001/XInclude"
+XI_INCLUDE = "{%s}include" % XI_URL
+
+VPR_TILE_PREFIX = 'BLK-TL-'
```

Comment:
Since we now have metadata, we were moving away from having this information in the names.

Reply:
It's never been about the metadata, it about segregation of namespaces.